test/test_user_interface.py

```diff
@@ -0,0 +1,17 @@
+# TODO the following need tests
+#   pack
+#   create_and_pack_label
+#   PickerWidget
+#   FolderPickerWidget
+#   FilePickerWidget
+#   CheckboxWidget
+#   SelectWidget
+#   FormVariantSelection
+#   InputFolderPickerWidget
+#   OutputFolderPickerWidget
+#   AnswerKeyPickerWidget
+#   ArrangementMapPickerWidget
+#   ProgressTrackerWidget
+#   create_and_pack_progress
+
+#   MainWindow
```

Comment:
I'd rather not merge TODO comments into the codebase, since they don't really add much value. At the same time, I totally agree we need these tests. Instead of the comment, could you create a new issue for adding UI tests?